content/articles/signal-handling-in-c++/index.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /signal-handling-in-cpp/
+title: Signal Handling In C++
+description: This article will provide an overview of signals and how they can be handled in C++.
+author: felix-maina
+date: 2021-12-16T00:00:00-13:30
+topics: [Security]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/signal-handling-in-cpp/hero.jpg
+    alt: Signal Handling In C++
+---
+There are interruptions that come up in your program, which are delivered to a process that runs on an operating system and pauses the running process. These interrupts are called signals.
+<!--more-->
+The signals can be generated by pressing `Ctrl+C` on various operating systems (Windows, Mac OS X, LINUX, or UNIX).
+
```

Comment:
The keys combination `CTRL + C` sends an interrupt signal on most cases. Please mention this.